projects/plugins/jetpack/extensions/blocks/videopress/deprecated/v2/utils.js

```diff
@@ -25,31 +24,29 @@ export const ASPECT_RATIOS = [
  */
 export const filterVideoPressClasses = ( className, videoPressClassNames ) => {
 	// Filter out VideoPress specific styles from className attribute.
-	const aspectRatioClassNames = ASPECT_RATIOS.reduce(
-		( accumulator, { className: ratioClass } ) => {
-			accumulator[ ratioClass ] = false;
-			return accumulator;
-		},
-		{ 'wp-has-aspect-ratio': false }
-	);
 
-	const customClasses = classnames( className, {
+	const aspectRatioClassNames = ASPECT_RATIOS.reduce( ( accumulator, ratio ) => {
+		accumulator.push( ratio.className );
+		return accumulator;
+	}, [] );
```

Comment:
If you want you could simplify this a bit
```suggestion
	const aspectRatioClassNames = ASPECT_RATIOS.map( ratio => ratio.className );
```

Reply:
Ha. Clearly I wasn't thinking.